portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tony Shen - Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <!--Google Font API-->
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto&effect=anaglyph">
+        <script src="script.js"></script>
+    </head>
+    <body style="page-container">
+        <div class="axis-0-grid-container">
+            <div class="header-container">
+
+                <div class="logo-section" href="index.html">
+                    <img class="logo" src="images/profile.jpg"/>
+                </div>
+
+                <div class="font-effect-anaglyph title-section"><a href="index.html">Tony Shen</a></div>
+
+                <div class="navigator-section">  
```

Comment:
There is a lot of code repetition in this site which would be good to address. It creates code bloat and can lead to errors. For example, you have some references to blog.html labelled as Blog and some as Vlog.

Surprisingly, it looks like there is no way to deal with it in pure HTML (https://css-tricks.com/the-simplest-ways-to-handle-html-includes/). In the old days I would use iframes, but that's probably even worse than duplication.

I think it worth to discuss within the pod, or with your virtual support. If there is no easy solution we can submit as is.

Reply:
Thanks for the comment! I noted this problem as well. I will try to reach out the virtual support engineering about a solution to this (Haven't got an invite to the chat group yet). Meanwhile, I will try one of the method linked?